projects/rancher/local-path-provisioner/Makefile

```diff
@@ -1,6 +1,6 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
-GOLANG_VERSION?="1.13"
+GOLANG_VERSION?="1.18"
```

Comment:
Building with 1.18 since kind is being their version with 1.18. The upstream local-path project seems to be using 1.16 still.